documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -6,10 +6,42 @@
 [id='assembly-upgrade-{context}']
 = Upgrading Strimzi
 
-Strimzi can be upgraded from a lower version to a higher version to take advantage of new features and enhancements, performance improvements, and security options.
+Strimzi can be upgraded from a given version to a later version to take advantage of new features and enhancements, performance improvements, and security options.
```

Comment:
I wonder if we should really use here the actual version number ... since as this stands, it applies for Strimzi 0.22.0. It does not apply to 0.21 if upgrading from 0.19. And we do not know how it applies to 0.23 in the future (I hope it will, but 0.23 will have its own docs with possible changes if there are any). Maybe that is why the `later` is so confusing for me.

Reply:
Agreed. I've changed this to use the actual version number.

> Strimzi can be upgraded to version {ProductVersion} to take advantage of new features and enhancements, performance improvements, and security options.